src/App.vue

```diff
@@ -37,7 +37,7 @@
 
       <v-container>
         <v-alert :value="!!message" type="error" dismissible>
-          <span v-html="message" />
+          <span>{{ message }}</span>
```

Comment:
Does `message` never need HTML content?

Reply:
I honestly don't know, but I suppose it is probable as v-html was used. We may need to use a sanitizing library to get rid of this warning.

Reply:
Is there a sanitizing library that's smarter than what Vue uses internally? Might have to investigate the possible sources of data that get pumped into this `message` parameter. Not necessarily today, but it would be good to know.